Documentation/Installation/Deployer/Index.rst

```diff
@@ -1,17 +1,90 @@
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
-.. index:: Deployment; Deployer
+..  index:: Deployment; Deployer
 
-.. _deployment-deployer:
+..  _deployment-deployer:
 
 ========
 Deployer
 ========
 
-`Deployer <https://deployer.org>`__ is a deployment tool
-written in PHP. A full description about how to get deployer running
-for TYPO3 can be found at https://t3terminal.com/blog/typo3-deploy/
+`Deployer <https://deployer.org/>`__ is a deployment tool written in PHP. Please
+also have a look into the
+`documentation of Deployer <https://deployer.org/docs/7.x/installation>`__,
+mainly the "Basics" and "Main Concepts".
 
-..  literalinclude:: _deploy.php
+..  todo: Explain:
+    * releases (which are symlinked)
+
+.. contents:: **Table of Contents**
+   :local:
+
+
+Installation
+============
+
+It is recommended to install Deployer either as a separate project or in a
+sub-folder of your project, for example :file:`<project-root>/deployment/`. This
+omits dependency problems with your project's dependencies.
+
+..  code-block:: bash
+
+    composer require deployer/deployer
+
+
+Initialize configuration
+========================
+
+Afterwards you can initialize the configuration with
+
+..  code-block:: bash
+
+    vendor/bin/dep init
+
+On console you are asked interactively some questions to generate a
+configuration:
+
+#.  Recipe language: The following sections use the PHP configuration, so choose
+    `php`.
+
+#.  Project template: You may choose `typo3` as project template, but this is
+    slightly outdated, so it is suggested to use the `common` template.
+
+#.  Repository: Enter the repository URL of your TYPO3 project.
```

Comment:
This about a git repository? How would one enter the URL, using https:// or ssh://, an example might help to understand.
If auto-detection is used, maybe mention this here?